model/project.go

```diff
@@ -68,6 +68,9 @@ type Project struct {
 
 	// Flag that indicates a project as requiring user authentication
 	Private bool `yaml:"private,omitempty" bson:"private"`
+
+	// Number of includes in the project cached for validation
+	NumIncludes int
```

Comment:
Typically in marshalling/unmarshalling between different data formats (YAML, BSON, etc.), the marshalling logic will try to use any exported field with a default field name as long as we haven't explicitly told it to ignore the field. From [the BSON marshalling docs](https://pkg.go.dev/go.mongodb.org/mongo-driver@v1.13.1/bson#hdr-Structs):

>1. Only exported fields in structs will be marshalled or unmarshalled.
>2. When marshalling a struct, each field will be lowercased to generate the key for the corresponding BSON element. For example, a struct field named "Foo" will generate key "foo". This can be overridden via a struct tag (e.g. `bson:"fooField"` to generate key "fooField" instead). 

Since we don't plan on persisting this field, we should use `"-"` for the struct tags because we don't want this field to persist:
```go
NumIncludes int `yaml:"-" bson:"-"`
```

Reply:
Thank you for explaining!